### PR TITLE
Add R-Grid component

### PR DESCRIPTION
**Pull request by anthonylebrun**
### What was a problem?

Recomponent needs a grid component.

### How this PR fixes the problem?

It's a grid component.

### Check lists

- [x] Readme file updated with actual description and example
- [x] Added all ARIA attributes required for component
- [ ] Added tests for both browser and SSR render and for all components properties
- [x] Added story with all knobs and actions

### Additional Comments

WIP - still need to review API and have some open questions about styles. Could use some assistance on the SSR stuff, as I'm not super familiar with that.

### CODE REVIEW

**Comment by anthonylebrun** [comment]
Hmm that's weird, the failures are lint failures that are not related to this branch ^. Was CI linting only added recently? cc @followyouhome

File: packages/recomponents/src/components/r-repeater/r-repeater.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Repeater
+
+This is an abstract component to help you render data in a naked table. It is notably used by `r-grid` under the hood, which is probably the component you want to use :)
```
Comment (mathewsonke) [inline packages/recomponents/src/components/r-repeater/r-repeater.md]: This isn't strictly correct, I believe `r-repeater` is intended to be used on its own outside of Grid

File: packages/recomponents/src/components/r-repeater/r-repeater.vue [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            },
+        },
+        created() {
+            this.fetchData();
+        },
+    };
+</script>
+
+<style>
+    @import '../../styles/table.scss';
```
Comment (mathewsonke) [inline packages/recomponents/src/components/r-repeater/r-repeater.vue]: I think we'll want to use absolute paths here

File: packages/recomponents/src/index.js [hunk 3]
```diff
@@ -45,16 +47,27 @@ const components = {
...
 
-function install(Vue) {
+function install(Vue, options = {}) {
     Vue.use(VueI18n);
 
+    /**
+     * Set global settings
+     */
+
+    Vue.RecomponentsSettings = {
```
Comment (mathewsonke) [inline packages/recomponents/src/index.js]: This should be namespaced, maybe something like `Vue.$recomponents.settings`

**Comment by anthonylebrun** [comment]
@mathewsonke Thanks, glad for the early feedback too :)

File: packages/recomponents/src/components/r-repeater/r-repeater.vue (hunk 2 above)
Comment (anthonylebrun) [reply packages/recomponents/src/components/r-repeater/r-repeater.vue]: Agreed, but the default alias doesn't seem to be working atm. Need to dig into that.

Comment (anthonylebrun) [reply packages/recomponents/src/components/r-repeater/r-repeater.vue]: cc @followyouhome ^

**Comment by anthonylebrun** [comment]
Is there anything special that we need to check for for SSR @followyouhome ?

**Comment by anthonylebrun** [comment]
@andrey-rebilly has agreed to take over this PR since leaving on PTO next week. Thanks Andrey!

File: packages/recomponents/src/components/r-repeater/r-repeater.vue (hunk 2 above)
Comment (followyouhome) [reply packages/recomponents/src/components/r-repeater/r-repeater.vue]: Should be done

**Pull request merged**